### PR TITLE
Update team page

### PR DESCRIPTION
**Pull request by dem4ron**
- Add new team-members
- Delete unused code
- Update Aron's bio
- Update bio

### CODE REVIEW

**Comment by dem4ron** [comment]
Hey @jonathandmiddleton, @LorettaExercism!
You can comment here your short bios and I'll update it on the site.

**Comment by LorettaExercism** [comment]
Loretta is the Head of Partnerships and Fundraising, with a track record of successful strategic and operational delivery across complex charities, including Cancer Research UK, UNICEF, Action For Children and NSPCC. She led the strategic development of fundraising at Abertay University.  She has significant experience of Learning and Development and is a qualified business coach with a passion for developing fundraisers skills.
Loretta is highly motivated by the transformational power of fundraising.

File: app/views/about/team.html.haml [hunk 1]
```diff
@@ -172,6 +198,27 @@
             .details
               %h3 Charles Care
               %p Charles was part of the team that built Exercism v2, working specifically on the server-side code and infrastructure. He helped build out the first version of our automated mentoring platform, and Github integration.
+          .person
+            = image_tag "team/karlo.jpg", alt: "An image of Karlo Soriano", class: "avatar"
+            .details
+              %h3
+                = link_to "Karlo Soriano", "https://twitter.com/kntsoriano"
+              .handle @kntsoriano
+              %p Karlo works on the website part of Exercism. He has created most of the website's interactive frontend content in React, and contributed to much of the Rails backend platform. In his spare times he is a semi-professional drummer and music producer. Karlo is also part of the team at #{link_to 'Kaido', 'https://kaido.org'}.
```
Comment (iHiD) [inline app/views/about/team.html.haml]:
```suggestion
              %p Karlo worked on the website part of Exercism. He created most of the website's interactive frontend for v3, and contributed to much of the Rails backend platform.
```

File: app/views/about/team.html.haml [hunk 2]
```diff
@@ -172,6 +198,27 @@
...
+                    %span Karlo's personal site
+                %li
+                  = link_to "https://github.com/kntsoriano" do
+                    = graphical_icon "external-site-github"
+                    %span GitHub
+
+                %li
+                  = link_to "https://twitter.com/kntsoriano" do
+                    = graphical_icon "external-site-twitter"
+                    %span Twitter
```
Comment (iHiD) [inline app/views/about/team.html.haml]:
```suggestion
```

**Comment by iHiD** [comment]
@dem4ron You need a proper faceshot here - sorry! :)

File: app/views/about/team.html.haml [hunk 3]
```diff
@@ -96,62 +96,88 @@
...
                     = graphical_icon "external-site-github"
                     %span GitHub
-
+          .person
+            = image_tag "team/loretta.jpg", alt: "An image of Loretta Murray", class: "avatar"
+            .details
+              %h3
+                = link_to "Loretta Murray", "https://twitter.com/LorettaBMThink", target: "_blank", rel: 'noopener'
+              .handle @LorettaExercism
+              %p Loretta is the Head of Partnerships and Fundraising, with a track record of successful strategic and operational delivery across complex charities, including Cancer Research UK, UNICEF, Action For Children and NSPCC. She led the strategic development of fundraising at Abertay University. She has significant experience of Learning and Development and is a qualified business coach with a passion for developing fundraisers skills. Loretta is highly motivated by the transformational power of fundraising.
```
Comment (iHiD) [inline app/views/about/team.html.haml]:
```suggestion
              %p Loretta is highly motivated by the transformational power of fundraising, and has a track record of successful strategic and operational delivery across complex organisations, including Cancer Research UK, UNICEF, NSPCC. and Abertay University. She is a qualified business coach with a passion for developing fundraisers skills.
```

**Comment by jonathandmiddleton** [comment]
@dem4ron 

Jonathan is the Community Manager at Exercism! He loves connecting with people and hearing stories of how people are where they are in life.  He is curious to learn and has been slowly getting to grips with programming.  He loves the outdoors and has a young family who live in the world's best city...Cape Town, South Africa.  
Seeing people gain access to skills that empower and improve life is what drives him every day!

**Pull request merged**